src/memote/checks/Basic.jl

```diff
@@ -0,0 +1,96 @@
+"""
+$(TYPEDEF)
+
+Basic metabolic model tests.
+"""
+module Basic
+
+using DocStringExtensions
+using COBREXA
+import ..Config
+
+"""
+$(TYPEDSIGNATURES)
+
+Test if the model has an id. Note, this test will fail unless you use a
+`StandardModel` and its id field is assigned.
+"""
+model_has_name(model) = begin
+    model isa StandardModel && model.id != "" && return true
+    return false # TODO need accessors for the model name
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+Test if the model has any metabolites.
+"""
+model_has_metabolites(model) = n_metabolites(model) > 0
+
+"""
+$(TYPEDSIGNATURES)
+
+Test if the model has any reactions.
+"""
+model_has_reactions(model) = n_reactions(model) > 0
+
+"""
+$(TYPEDSIGNATURES)
+
+Test if the model has any genes.
+"""
+model_has_genes(model) = n_genes(model) > 0
+
+"""
+$(TYPEDSIGNATURES)
+
+Calculate the metabolic coverage by dividing the number of reactions by the
+number of genes.
+"""
+model_metabolic_coverage(model) = n_reactions(model) / n_genes(model)
```

Comment:
is it just me or this just throws error if there are no genes?

Reply:
this would divide by zero if there are no genes, but no genes =  truly terrible model